src/main/java/hudson/plugins/report/jck/AbstractReportPublisher.java

```diff
@@ -83,7 +83,7 @@ final public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, Build
         List<Suite> report = build.getWorkspace().act(
                 new ReportParserCallable(reportFileGlob, createReportParser()));
         if (report.stream().anyMatch(
-                s -> s.getReport() != null && (s.getReport().getTestsError() != 0 || s.getReport().getTestsFailed() != 0 || s.getReport().getTestsTotal() <= 0))) {
+                s -> s.getReport() != null && (s.getReport().getTestsError() != 0 || s.getReport().getTestsFailed() != 0) || s.getReport().getTestsTotal() <= 0)) {
```

Comment:
You forget to revert my initial commit. 
```
s.getReport().getTestsTotal() <= 0
```
Do not belong here

Reply:
fixed